### DISCUSSION TITLE
Welcome to twitch-poll-overlay Discussions!

### DISCUSSION THREAD

**Discussion by darmiel** [darmiel/twitch-poll-overlay#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by napoellis** [comment]
Does this poll overlay still work? And is there a way to use text as an option for the poll votes instead of numbers
